packages/plugin-selenium-driver/src/plugin/index.ts

```diff
@@ -421,358 +429,301 @@ export class SeleniumPlugin implements IBrowserProxyPlugin {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return (await client.$(xpath)).clearValue();
-        }
+        const selector = await client.$(xpath);
+        return selector.clearValue();
     }
 
     public async keys(applicant: string, value: any) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return client.keys(value);
-        }
+        return client.keys(value);
     }
 
     public async elementIdText(applicant: string, elementId: string) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return client.getElementText(elementId);
-        }
+        return client.getElementText(elementId);
     }
 
     public async elements(applicant: string, xpath: string) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            const elements = await client.findElements('xpath', xpath) as unknown;
-            const a = (elements as Array<Record<string, string>>).map((o) => {
-                const keys = Object.keys(o);
-                return { 'ELEMENT': o[keys[0]] };
-            });
-
-            return a;
-        }
+        const elements = await client.findElements('xpath', xpath) as unknown;
+        return (elements as Array<Record<string, string>>).map((o) => {
+            const keys = Object.keys(o);
+            return { 'ELEMENT': o[keys[0]] };
+        });
     }
 
     public async frame(applicant: string, frameID: string) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return client.switchToFrame(frameID);
-        }
+        return client.switchToFrame(frameID);
     }
 
     public async frameParent(applicant: string) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return client.switchToParentFrame();
-        }
+        return client.switchToParentFrame();
     }
 
     public async getValue(applicant: string, xpath: string) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return (await client.$(xpath)).getValue();
-        }
+        const selector = await client.$(xpath);
+        return selector.getValue();
     }
 
     public async keysOnElement(applicant: string, xpath: string, value: any) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return (client as any).keysOnElement(xpath, value);
-        }
+        return (client as any).keysOnElement(xpath, value);
```

Comment:
`any` still needed?

Reply:
Yes. But we can describe our interface of BrowserObject.